src/store/reducers/taskReducer.jsx

```diff
@@ -0,0 +1,13 @@
+import { ADD_TASK } from "src/store";
+
+const initialState = [];
+const taskReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_TASK:
+      return [action.payload, ...state];
```

Comment:
Are we maintaining[ redux-immutability](https://redux.js.org/usage/structuring-reducers/immutable-update-patterns) pattern here ?